app/assets/javascripts/govuk_publishing_components/components/feedback.js

```diff
@@ -74,6 +76,28 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       this.setHiddenValuesNotUsefulForm(gaClientId)
     }.bind(this))
 
+    this.somethingIsWrongButton.addEventListener('click', function (e) {
```

Comment:
I like that the timer isn't initialised until the user clicks the button to expand the form, it fits with only starting once the user is actually using the form but also means that there's no interval happening on every page on GOV.UK unless it's actually needed. 

Would it be possible to similarly limit the other listeners (paste and keypress), perhaps by initialising them inside this click listener as well?